pages/workflows/[id].tsx

```diff
@@ -7,7 +7,7 @@ import Layout from "../../components/layout";
 import WorkflowTags from "../../components/WorkflowTags";
 import { Argument, Workflow } from "warp-workflows";
 import { CopyIcon } from "../../components/icons/copy";
-import ReactTooltip from "react-tooltip";
+import { Tooltip } from "react-tooltip";
```

Comment:
Yet another major version upgrade with a slightly different API. See the [upgrade guide](https://react-tooltip.com/docs/upgrade-guide/changelog-v4-v5) for more details on these changes.